training_pipeline/pipeline/configs.py

```diff
@@ -42,6 +43,59 @@
 EXAMPLE_GEN_BEAM_ARGS = None
 TRANSFORM_BEAM_ARGS = None
 
+"""
+EVAL_CONFIGS is to configuration for the Evaluator component
+to define how it is going to evalua the model performance.
+
+tfma.ModelSpec
+    signature_name is one of the signature in the SavedModel from
+    Trainer component, and it will be used to make predictions on 
+    given data. preprocessing_function_names allows us to include
+    a set of transformation(preprocessing) signatures in the Saved
+    Model from Trainer component. 
+
+    label_key and prediction_key will be used to compare the ground
+    truth and prediction results.
+
+slicing_specs
+    we use the entire dataset to evaluate the model performance. If
+    you want to evaluate the model based on different slices of data
+    set, you should prepare TFRecords to have multiple features which
+    of each corresponds to each slices(or categories), then write the
+    slicing_specs options accordingly.
+"""
+EVAL_CONFIGS = tfma.EvalConfig(
+    model_specs=[
+        tfma.ModelSpec(
+            signature_name="from_examples",
+            preprocessing_function_names=["transform_features"],
+            label_key="label_xf",
+            prediction_key="label_xf",
+        )
+    ],
+    slicing_specs=[tfma.SlicingSpec()],
```

Comment:
Does it denote default behaviour? If so, what is the default behviour?

Reply:
Actually, there is no default. If there is default, we don't have to specify things explicitly. However, default here means that it does not have special behaviours on `Slicing` that evaluates the model on the given dataset with one slice. 

each slice could mean different feature. in order to evaluate the model performance for different features differently, then TFRecords should hold the data accordingly. 
- for instance, instead of `image`, we could save different categories from the `image` into `street`, `building`, ....

Reply:
Okay. Thanks for explaining. Very interesting.

So, the current configuration will only take the current eval dataset as is treating it to be a single slice?

Reply:
Yup